docs/resources/conformity_communication_settings.md

```diff
@@ -99,6 +99,7 @@ Other arguments:
      * `categories` (Array of String) - (Optional) An array of category (AWS well-architected framework category) strings from the following:
           security | cost-optimisation | reliability | performance-efficiency | operational-excellence
      * `compliances` (Array of String) - (Optional) An array of supported standard or framework ids. Possible values: ["AWAF" | "CISAWSF" | "CISAZUREF" | "CISAWSTTW" | "PCI" | "HIPAA" | "GDPR" | "APRA" | "NIST4" | "SOC2" | "NIST-CSF" | "ISO27001" | "AGISM" | "ASAE-3150" | "MAS" | "FEDRAMP"]
+     * `statuses` (only used for SNS and webhook channels) An array of statuses strings from the following: SUCCESS | FAILURE
```

Comment:
⛏️ We should move this to between `services` and `tags` so that it stays alphabetical 😄